app/controllers/offices_controller.rb

```diff
@@ -42,8 +42,10 @@ def update
     office.assign_attributes(office_params)
     authorize office
 
-    if office.save && manager_setup.in_progress?
-      redirect_to out_of_the_box_redirect
+    if office.save
```

Comment:
Just curious as to what happens when a manager signs in for a first time?

Before they'd be prompted to assign certain offices, but what happens now?